crates/turborepo-lib/src/cli/mod.rs

```diff
@@ -1156,15 +1160,20 @@ pub async fn run(
             config::run(base).await?;
             Ok(0)
         }
-        Command::Ls { filter, packages } => {
+        Command::Ls {
+            affected,
+            filter,
+            packages,
+        } => {
             warn!("ls command is experimental and may change in the future");
             let event = CommandEventBuilder::new("info").with_parent(&root_telemetry);
 
             event.track_call();
+            let affected = *affected;
```

Comment:
`ls` currently suffers from the same "double run arg" problem. Doesn't need to happen in this PR, but there should be checks to make sure `cli_args.execution_args.affected` and `cli_args.execution_args.filter` aren't defined.